### PR TITLE
windows.open noopener

### LINKED ISSUE #6862: [Bug] Space shotcut get white background before redirect
### Describe the bug

I am using docker version.

If i use shotcut "space" to open in new tab, it will first open a white background page before redirect, but if use mouse left click title, it become dark background page.

Browser use dark mode, other page is normal dark background.

Use "space" shotcut:
image

Use left click title:
image

### To Reproduce

Use "space" to open a new tab.

### Expected behavior

As same as left click

### FreshRSS version

1.24.3

### Environment information

Docker image: freshrss/freshrss:latest

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Alkarex**
fix https://github.com/FreshRSS/FreshRSS/issues/6862
Rework https://github.com/FreshRSS/FreshRSS/pull/2506

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]
Nothing to say about it. ;-)

**Comment by czf1009** [comment]
Updated the docker image and it worked! Thanks a lot!

File: p/scripts/main.js [hunk 1]
```diff
@@ -1099,10 +1099,8 @@ function init_shortcuts() {
 
 			const link_go_website = document.querySelector('.flux.current a.go_website');
 			if (link_go_website) {
-				const newWindow = window.open();
+				const newWindow = window.open(link_go_website.href, '_blank', 'noopener');
 				if (newWindow) {
-					newWindow.opener = null;
-					newWindow.location = link_go_website.href;
 					ev.preventDefault();
```
Comment (Frenzie) [inline p/scripts/main.js:1104]: There's a regression hiding in here. newWindow is `null`, just as if it was blocked. That being said, it probably makes sense to do preventDefault without any conditions regardless? As a user I'm not expecting it to do whatever the default browser action is, but for it to be overridden as per the settings.

image

Comment (Alkarex) [reply p/scripts/main.js:1104]: Do you plan to send a PR?

Comment (Frenzie) [reply p/scripts/main.js:1104]: https://github.com/FreshRSS/FreshRSS/pull/7089

**Pull request merged**